src/WebInterface/Presentation/Http/View/base.html.twig

```diff
@@ -3,18 +3,16 @@
 <head>
     <title>Gaming</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no"/>
-    <link rel="apple-touch-icon" sizes="180x180" href="{{ asset('assets/img/icon/apple-touch-icon.png') }}"/>
-    <link rel="icon" type="image/png" sizes="32x32" href="{{ asset('assets/img/icon/favicon-32x32.png') }}"/>
-    <link rel="icon" type="image/png" sizes="16x16" href="{{ asset('assets/img/icon/favicon-16x16.png') }}"/>
-    <link rel="manifest" href="{{ asset('assets/img/icon/site.webmanifest') }}"/>
-    <link rel="mask-icon" href="{{ asset('assets/img/icon/safari-pinned-tab.svg') }}" color="#427280"/>
+    <link rel="apple-touch-icon" sizes="180x180" href="{{ asset('icon/apple-touch-icon.png') }}"/>
+    <link rel="icon" type="image/png" sizes="32x32" href="{{ asset('icon/favicon-32x32.png') }}"/>
+    <link rel="icon" type="image/png" sizes="16x16" href="{{ asset('icon/favicon-16x16.png') }}"/>
+    <link rel="manifest" href="{{ asset('icon/site.webmanifest') }}"/>
+    <link rel="mask-icon" href="{{ asset('icon/safari-pinned-tab.svg') }}" color="#427280"/>
     <meta name="msapplication-TileColor" content="#da532c"/>
     <meta name="theme-color" content="#ffffff"/>
-    <link rel="stylesheet" href="{{ asset('assets/css/main.css') }}"/>
-    <script type="text/javascript">
```

Comment:
Removing those polyfills means that older browsers aren't supported anymore. The readme should be updated to mention that.